tests/test_pyscalcli.py

```diff
@@ -73,9 +73,7 @@ def test_pyscal_client_static(tmp_path, caplog, default_loglevel, mocker):
     assert Path("relperm.inc").is_file()
     assert not any(record.levelno == logging.ERROR for record in caplog.records)
     assert not any(record.levelno == logging.INFO for record in caplog.records)
-
-    # We get one warning due to empty cells in xlsx:
-    assert sum(record.levelno == logging.WARNING for record in caplog.records) == 1
+    assert sum(record.levelno == logging.WARNING for record in caplog.records) == 0
```

Comment:
(I am not sure whether this line just be deleted altogheter or not)

Reply:
Passed the test after deleting that line. Will delete it from the final version as well.

Reply:
ok!